providers/aws/resources/aws.lr

```diff
@@ -967,15 +967,15 @@ private aws.sagemaker.notebookinstance @defaults("arn name") {
   // Name of the notebook instance
   name string
   // Details about the notebook
-  details() aws.sagemaker.notebookinstance.details
+  details() aws.sagemaker.notebookinstancedetails
```

Comment:
this might be a breaking change for stored assets/resolved policies, im not 100% though

Reply:
chatted with jay in discord, he said this would get fixed on re-scan if it was an issue, so this is a non-breaking change